max_search_radius: 20.0
max_rotation_speed: 0.3
max_speed: 1.0
target_duration: 10.0
publish_rate: 10.0
buffer_length: 10.0
max_cluster_length: 5.0
min_cluster_length: 0.3
map_frame: world
robot_frame: base_link